### ISSUE TITLE
When run on travis, doesn't this automatically fix errors?

### ISSUE THREAD

**Issue by scytacki** [concord-consortium/portal-pages#60]
When run on travis, doesn't this automatically fix errors?

https://github.com/concord-consortium/portal-pages/blob/128a4776913d25ec394c4502de87a33e3edbb318/package.json#L9

I don't think that is what we want to do on travis. What do you thing @pjanik

**Comment by kswenson** [comment]
I would tend to agree. In my projects I've generally had the regular `lint` script not fix errors and then added a separate `lint:fix` script which did.

**Comment by pjanik** [comment]
Yes, good catch, I haven't thought about it (as I mostly run lint myself and not all project run it on Travis). I agree, makes more sense to run lint without --fix on Travis.
